code/src/components/TaskList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import 'components/TaskList.css'
+import { Task } from 'components/Task'
+
+
+export const TaskList = () => {
+
+    const allTasks = useSelector((state) => state.tasks.items)
+    console.log(allTasks)
+
+
+
+    return (
+        <div>
+
+            <ul className="task-list-container">
+                {allTasks.map((task) => (
+                    <Task key={task.id} task={task} />
+                ))}
```

Comment:
And here map through both of those arrays

```
{uncompletedTasks.map((task) => (
  <Task key={task.id} task={task} />
))}
{completedTasks.map((task) => (
  <Task key={task.id} task={task} />
))}
```

Reply:
then the uncompleted tasks will be listed on top and then below comes the completed ones.